gnmi_server/server.go

```diff
@@ -166,15 +184,16 @@ func NewServer(config *Config, opts []grpc.ServerOption) (*Server, error) {
 	if err != nil {
 		return nil, fmt.Errorf("failed to open listener port %d: %v", srv.config.Port, err)
 	}
-	gnmipb.RegisterGNMIServer(srv.s, srv)
-	spb_jwt_gnoi.RegisterSonicJwtServiceServer(srv.s, srv)
+	gnmipb.RegisterGNMIServer(srv.s, srv.gnmiServer)
+	spb_jwt_gnoi.RegisterSonicJwtServiceServer(srv.s, srv.gnoiServer)
 	if srv.config.EnableTranslibWrite || srv.config.EnableNativeWrite {
-		gnoi_system_pb.RegisterSystemServer(srv.s, srv)
+		gnoi_system_pb.RegisterSystemServer(srv.s, srv.gnoiServer)
+		gnoi_file_pb.RegisterFileServer(srv.s, srv.gnoiServer)
```

Comment:
Please create systemserver and fileserver to replace this gnoi server

Reply:
Thanks, changed to systemServer and fileServer